diff --git a/node_modules/react-native-video-trim/ios/VideoTrim.swift b/node_modules/react-native-video-trim/ios/VideoTrim.swift
index 1af0d68..04a8392 100644
--- a/node_modules/react-native-video-trim/ios/VideoTrim.swift
+++ b/node_modules/react-native-video-trim/ios/VideoTrim.swift
@@ -545,7 +545,7 @@ public class VideoTrim: RCTEventEmitter, AssetLoaderDelegate, UIDocumentPickerDe
                       
                       // Handle removeAfterSavedToPhoto
                       if let removeAfterSaved = config["removeAfterSavedToPhoto"] as? Bool, removeAfterSaved {
-                        let _ = VideoTrim.deleteFile(url: outputFile.absoluteString)
+                        let _ = VideoTrim.deleteFile(url: outputFile)
                       }
                       
                       let result = [
